README.md

```diff
@@ -52,7 +52,7 @@ Output format can take several forms:
  * `Plaintext` : raw text (php object, as returned by print_r)
  
 
-**Shipped version:** 2021-04-25~ynh1
+**Shipped version:** 2021.04.25~ynh1
```

Comment:
Why differing from the [official version naming](https://github.com/RSS-Bridge/rss-bridge/releases)...? Is there a technical reason?

Reply:
the regex for checking the app version don't accept `-` cf: https://ci-apps.yunohost.org/ci/job/5003

Reply:
OK, then the [version pattern](https://github.com/pypa/packaging/blob/20cd09e00917adbc4afeaa753be831a6bc2740f7/packaging/version.py#L225) is faulty here... But given the complexity of it, let's not break it! 😄

Reply:
So what should we do ? I don't get it :sweat_smile:

Reply:
I guess this merge means "we keep it as it is" ^^

Reply:
> I guess this merge means "we keep it as it is" ^^

yes, This is mainly to keep the CI happy and to bring back the app to level 8